.gitpod.yml

```diff
@@ -1,9 +1,11 @@
 tasks:
-  - init: |
-      mvn install -P quick-build
+  - init: mvn install -P quick-build
```

Comment:
@akosyakov not sure if you will know but for some reason I get an internal server error on preview:

> HISTFILE=/workspace/.gitpod/cmd-1 history -r; {
> [ -r /workspace/.prebuild-log-1 ] && cat /workspace/.prebuild-log-1; [ -r /workspace/.gitpod/prebuild-log-1 ] && cat /workspace/.gitpod/prebuild-log-1; true
> } && {
> gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
> }
Awaiting port 8080... ok
https://8080-jenkinsci-designlibrary-cvykmhpwggp.ws-eu45.gitpod.io
2022/05/18 17:51:27 500 Internal Server Error


<img width="1299" alt="image" src="https://user-images.githubusercontent.com/21194782/169110916-3a026bd7-3e2d-4e09-9ae6-f6f2031c018e.png">

Copying the command into my terminal and running it is just fine:

>  gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/

Reply:
hm, it is probably bug on our side we need to wait till JB backend is running.

Could you try to add  `gp await-port 63342`  as a workaround?

Reply:
that fixed it thanks!

Reply:
It worked the first time but failed the second time:

```
gitpod /workspace/design-library-plugin (timja-patch-1) $  HISTFILE=/workspace/.gitpod/cmd-1 history -r; {
> [ -r /workspace/.prebuild-log-1 ] && cat /workspace/.prebuild-log-1; [ -r /workspace/.gitpod/prebuild-log-1 ] && cat /workspace/.gitpod/prebuild-log-1; true
> } && {
> gp await-port 63342 && gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
> }
Awaiting port 63342... ok
Awaiting port 8080... ok
https://8080-jenkinsci-designlibrary-pigv2mnet01.ws-eu45.gitpod.io
2022/05/19 19:43:01 500 Internal Server Error

```

Reply:
We are looking in permanent fix, most likely will be out beginning next week.